gradle.properties

```diff
@@ -3,7 +3,7 @@ projectGroup=org.eclipse.dataspaceconnector.identityhub
 defaultVersion=0.0.1-SNAPSHOT
 
 edcGroup=org.eclipse.dataspaceconnector
-edcVersion=0.0.1-SNAPSHOT
+edcVersion=0.0.1-milestone-5
```

Comment:
@paullatzelsperger is this change needed, Because i see even on milestone-5 tag EDC package version is still 0.0.1-SNAPSHOT
 
https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/39e8da36860a9087eae76fcd3cdae2023b4e30d0/gradle.properties#L2

or maybe I am missing something, good to see your though on this. Thanks.

Reply:
the package version of EDC gets overridden during the release process, so yes, it is needed.

Reply:
edit: updated description to also inform about the removing of the local checkout/build